docs/user-guide/metadata.md

```diff
@@ -141,3 +143,24 @@ jobs:
 These settings will result in Git checks that look like:
 
 ![PR checks](./assets/pr-checks.png)
+
+### Coverage and Test Results
+
+You can populate coverage and test results on build page from a Screwdriver build through using metadata. Screwdriver UI will read from `tests.coverage` and `tests.results` in metadata and display them.
+
+Example screwdriver.yaml should look something like below:
+
+```
+jobs:
+  main:
+    steps:
+      - set-coverage-and-test-results: |
+          meta set tests.coverage  100 # this should be the coverage percentage number
+          meta set tests.results 10/10 # this should be `pass_tests_number/total_tests_number`
+```
+
+> Note: metadata will override SonarQube results
+
+These settings will result in build page that looks like:
+
+![coverage-meta](./assets/coverage-meta.png)
```

Comment:
did we lose the `%` sign in the UI for coverage data?

also, the screenshot has some cut off text at the top that looks kind of strange

Reply:
we will add `%` in UI.

Reply:
uploaded the wrong screenshot! fixed it lol